configs/build_config.py

```diff
@@ -155,6 +162,40 @@ def manifesting_keys(self):
     def tftp_keys(self):
         return frozenset(self._tftp_env)
 
+    @property
+    def golden_image(self):
```

Comment:
I like the property.  Again, is this the time to rename it to "golden_tarball" to reflect what is really being stored?